docs/api-reference.md

```diff
@@ -144,28 +145,47 @@ _Attribute names_
 
 _Functions_
 
-    String formatKeyValuePair(String key, String value);
+    String FormatKeyValuePair(String key, String value);
 
 Format the attributes as 'key=value'. By default, some values are escaped or wrapped in curly braces to follow the ODBC standard, but you can also do it here if needed.
 
     bool MatchesConnectionAttributes(Object attr, Object inKey);
 
 Invokes attribute matching code.
 
+    String GetPlatform();
+
+Returns the platform information as a string.
```

Comment:
Better list all possible values, 
- win
- mac
- linux

Reply:
Merged this one already, but will fix it before it is merged into the main docs branch.

Reply:
Updated function on this pull request: https://github.com/tableau/connector-plugin-sdk/pull/193